scout/server/blueprints/cases/controllers.py

```diff
@@ -1397,13 +1397,7 @@ def _matching_causatives(
                 The subset of all secondary findings found in default gene panels
         )
     """
-    matching_causatives_filter = list(
-        set(other_causatives_filter + other_causatives_in_default_panels_filter)
```

Comment:
In the specific case of the bug other_causatives_filter = [] because there are no panels in the safe search for the institute:

<img width="497" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/d5deaa89-fbc5-4842-83eb-eb0742bafa2e">

other_causatives_in_default_panels_filter is the genes in the case default panels, check here: https://github.com/Clinical-Genomics/scout/blob/413d8301c84b4d070578ab53b7aed6a3ccb3f093/scout/server/blueprints/cases/controllers.py#L389C34-L389C64

I believe that that function is returning the list of genes in the case default panels, which us too limiting for searching the matching causatives in all cases of the institute